configure.ac

```diff
@@ -408,4 +409,8 @@ AS_IF([test "x$LUAPC" != "x"],
   [AC_MSG_NOTICE([Lua support: $LUAPC])],
   [AC_MSG_NOTICE([Lua support: no])]
 )
+AS_IF(test "x$enable_experimental_gss_tsig" != "x"],
```

Comment:
Could you test for `= "xyes"`? Running `./configure --with-modules='' --with-dynmodules=''` yelds:

```
configure: GSS-TSIG support: yes
```

in the configure output.